tests/topotests/bgp_default_originate/test_bgp_default_originate_topo1_3.py

```diff
@@ -952,7 +960,7 @@ def test_verify_default_originate_after_BGP_and_FRR_restart_p2(request):
             dut="r2",
             routes=DEFAULT_ROUTES,
             expected_nexthop=DEFAULT_ROUTE_NXT_HOP_R1,
-            expected=False,
+            expected=True,
```

Comment:
"expected" is by default True. We could ignore passing expected=True.

Reply:
ok, this can be removed.

Reply:
I see many places having `expected=True`, not sure if we should be explicit here?

Reply:
Yes It has been added many places, but it is by-default True. 
We can ignore this comment for now.